CHANGES.txt

```diff
@@ -27,6 +27,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Update CacheDir to use uuid for tmpfile uniqueness instead of pid.
       This fixes cases for shared cache where two systems write to the same
       cache tmpfile at the same time because the happened to get the same pid.
+    - Added support for passing custom CacheDir derived classes to SCons to
+      to support MongoDB's use of a compressed CacheDir tool. Moved copy_from_cache
```

Comment:
Add new API call signature. Also remove "MongoDB's". And add line at end "Code contributed by MongoDB".  Also add this blurb to the RELEASE.txt new features section

Reply:
updated per suggestions in ce9ea25